extensions/udapdr/scripts/dsp_utilities/generate_ColBERTv2_zeroshot_results.py

```diff
@@ -0,0 +1,87 @@
+
+import numpy as np
+import os
+import torch
+import tqdm
+import time
+
+from colbert.infra import Run, RunConfig, ColBERTConfig
```

Comment:
A general comment here:
for the imports to work, they would have to be rooted in the corresponding `extensions.udapdr` directory, e.g. `from extensions.udapdr.scripts.colbert.infra import Run, RunConfig, ColBERTConfig` , here and elsewhere in the `udapdr` directory.